packages/cli/src/cli/cogify/action.cog.ts

```diff
@@ -96,6 +96,7 @@ export class ActionCogCreate extends CommandLineAction {
     // Output file exists don't try and overwrite it
     if (outputExists) {
       logger.warn({ targetPath }, 'CogCreate:OutputExists');
+      await this.checkJobStatus(job, logger);
```

Comment:
@blacha  I have added this check in here. In cases, partially processed file list never get to `CogCreate:JobComplete` to create 'ts_' and 'im_' records.